packages/divider/package.json

```diff
@@ -1,36 +1,35 @@
 {
-    "name": "@spectrum-web-components/rule",
+    "name": "@spectrum-web-components/divider",
     "publishConfig": {
         "access": "public"
     },
     "repository": {
         "type": "git",
         "url": "https://github.com/adobe/spectrum-web-components.git",
-        "directory": "packages/rule"
+        "directory": "packages/divider"
     },
     "bugs": {
         "url": "https://github.com/adobe/spectrum-web-components/issues"
     },
-    "homepage": "https://adobe.github.io/spectrum-web-components/components/rule",
+    "homepage": "https://adobe.github.io/spectrum-web-components/components/divider",
     "keywords": [
         "spectrum css",
         "web components",
         "lit-element",
         "lit-html"
     ],
-    "version": "0.5.0",
+    "version": "0.0.1",
     "description": "",
-    "main": "./src/index.js",
-    "module": "./src/index.js",
-    "types": "./src/index.d.ts",
```

Comment:
We lost the types entries in the new generation... that's something we'll need to add to all 5 packages and to the generator as well.